### ISSUE TITLE
Build for 5.1.0+trunk

### ISSUE THREAD

**Issue by shakthimaan** [OCamlPro-Iguernlala/ocplib-simplex#13]
The following changes to `ocplib-simplex.0.4` builds the package for `5.1.0+trunk`. Thanks to Florian Angeletti (@Octachron) for the fix.
```
diff --git a/src/core.ml b/src/core.ml
index 1814f7b..8e340d7 100644
--- a/src/core.ml
+++ b/src/core.ml
@@ -239,7 +239,7 @@ module Make
             let re, im = info.value in
             let comp_status = re_computed_status_of_info info in
             Format.fprintf fmt
-              "%a   [ %a == (%4a , %4a) ]   %a   (computed %s) (flag %s)@."
+              "%a   [ %a == (%a , %a) ]   %a   (computed %s) (flag %s)@."
               print_min_bound info
               Var.print x
               R.print re
@@ -296,7 +296,7 @@ module Make
 
       let print_matrix fmt env =
         MX.iter
-          (fun x (_, p) ->fprintf fmt "%4a = %a@." Var.print x P.print p)
+          (fun x (_, p) ->fprintf fmt "%a = %a@." Var.print x P.print p)
           env.basic
 
         let print result fmt env =
```

**Comment by Stevendeo** [comment]
I overlooked this issue while releasing 0.5, but the good news is the patch is no longer necessary. I managed to compile ocplib-simplex.0.5 (soon available on opam) with ocaml.5.1.0+trunk. If it works for you as well, I will close this issue.

**Comment by Halbaroth** [comment]
Actually, I have to apply this patch on the ocplib-simplex 0.4 in order to release the next version of Alt-Ergo.

**Comment by bclement-ocp** [comment]
Fixed by #18, thanks for the report
